GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿<div class="column-third">
+    <aside class="govuk-related-items" role="complementary">
+        <h3 class="heading-small" id="subsection-title">Related content</h3>
+        <nav role="navigation" aria-labelledby="subsection-title">
+            <ul class="font-xsmall">
+                <li>
+                    <a rel="external noopener track" target="_blank" data-track-category="Related Link Clicked" href="https://www.gov.uk/government/collections/gender-pay-gap-reporting">Gender pay gap reporting<span class="visually-hidden">(opens in a new window)</span></a>
```

Comment:
Is there a reason we are hard coding these? e.g. this one is Global.GpgGuidanceLink

Reply:
I didn't know that, thanks!